api/renewer.go

```diff
@@ -79,6 +80,10 @@ type RenewerInput struct {
 	// RenewBuffer is the size of the buffered channel where renew messages are
 	// dispatched.
 	RenewBuffer int
+
+	//Increment specifies the number of seconds that the lease TTL is pushed into the future.
+	//If not set, then the lease TTL may be pushed into the future by what the system default TTL is set to.
```

Comment:
There are a couple of misleading bits here. The increment doesn't push the ttl into the future, it specifies a desired ttl starting from now. If not set, the value used may, but may not, be the system default ttl. Even if set, it may not be honored.